app/controllers/development_controller.rb

```diff
@@ -16,7 +16,14 @@ def group_url(group)
 
   def setup_group
     cleanup_database
-    test_group
+    test_group.add_admin! emilio
```

Comment:
Er, this looks like `setup_group` is adding an extra admin, while `setup_group_with_multiple_coordinators` is just adding an extra member?